contents/blog/is-google-analytics-hipaa-compliant.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2022-21-02
+title: Is Google Analytics HIPAA compliant?
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/hipaa-compliant-product-analytics.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Privacy", "Guides"]
+---
```

Comment:
(All of this is personal opinion, don't take it to heart, it has _nothing_ to do with quality of the content):

I'm guessing this is a heavily SEO optimised post, and that's the goal?

I hope we don't publish something like this on HN, because if I saw this on HN, I'd flag the shiz out of it.

----

So, before I start dishing out opinions, could use some guidance on what kind of review are you looking for? :)

Reply:
Yeah, this will go nowhere near HN or any social for that matter.